spec/components/candidate_interface/gcse_grade_guidance_component_spec.rb

```diff
@@ -1,139 +1,80 @@
 require 'rails_helper'
 
 RSpec.describe CandidateInterface::GcseGradeGuidanceComponent do
-  context 'when the subject is maths' do
-    it 'displays the guidance around the expectation of providers' do
-      subject = 'maths'
+  subject(:subjects) { %w[english science maths] }
 
-      result = render_inline(described_class.new(subject, nil))
+  context 'when the qualification type is O level for any subject' do
+    let(:qualification_type) { 'gce_o_level' }
 
-      expect(result.text).to include(t('gcse_edit_grade.guidance.main'))
-    end
-
-    it 'does not display the guidance around triple science' do
-      subject = 'maths'
-
-      result = render_inline(described_class.new(subject, nil))
-
-      expect(result.text).not_to include(t('gcse_edit_grade.guidance.o_level_triple_gcse_science'))
-    end
-
-    it 'does not display the guidance around english literature and multiple english qualifications' do
-      subject = 'maths'
-
-      result = render_inline(described_class.new(subject, nil))
-
-      expect(result.text).not_to include(t('gcse_edit_grade.guidance.multiple_english_gcses.main'))
-      expect(result.text).not_to include(t('gcse_edit_grade.guidance.multiple_english_gcses.secondary'))
-    end
-  end
-
-  context 'when the subject is science' do
-    it 'displays the guidance around the expectation of providers' do
-      subject = 'science'
-
-      result = render_inline(described_class.new(subject, nil))
-
-      expect(result.text).to include(t('gcse_edit_grade.guidance.main'))
-    end
-
-    it 'does not display the guidance around english literature and multiple english qualifications' do
-      subject = 'science'
-
-      result = render_inline(described_class.new(subject, nil))
+    it 'displays the O level guidance' do
+      subjects.each do |s|
+        result = render_inline(described_class.new(s, qualification_type))
```

Comment:
Can the specs be updated to use meaninful variable names e.g.

```suggestion
      subjects.each do |subject|
        result = render_inline(described_class.new(subject, qualification_type))
```

Reply:
Thanks, fixed